internal/rbd/rbd_util.go

```diff
@@ -570,6 +570,30 @@ func deleteImage(ctx context.Context, pOpts *rbdVolume, cr *util.Credentials) er
 	return nil
 }
 
+func (rv *rbdVolume) getImageParent() (*rbdVolume, error) {
+	parentVol := rbdVolume{}
+	rbdImage, err := rv.open()
+	if err != nil {
+		return nil, err
+	}
+	parentInfo, err := rbdImage.GetParent()
+	if err != nil {
```

Comment:
```suggestion
if err != nil {
		// Caller should decide whether not finding
		// the parent is an error or not.
		if errors.Is(err, librbd.ErrNotFound) {
                       // parentVol.RbdImageName="" by default?
			return &parentVol, nil
		}
		return nil, err

}
parentVol.RbdImageName = parentInfo.Image.ImageName
parentVol.Pool = parentInfo.Image.PoolName
parentVol.Monitors = rv.Monitors
parentVol.conn = rv.conn.Copy()
return &parentVol, nil
```

Reply:
Thanks for the suggestion! Updated with changes,